ding/utils/data/dataset.py

```diff
@@ -328,9 +328,10 @@ class D4RLTrajectoryDataset(Dataset):
         },
     }
 
-    def __init__(self, dataset_path: str, context_len: int, rtg_scale: float) -> None:
-
-        self.context_len = context_len
+    def __init__(self, cfg: dict) -> None:
+        dataset_path = cfg.policy.collect.get('data_path', None)
```

Comment:
don't use `xxx.get`, we must ensure all the configs are fixed after `compile_config`